client/src/components/Settings/SettingsPrivacy.res

```diff
@@ -1,12 +1,19 @@
 // open Tc
 
+// TODO describe file/module
+
 module FullstoryJs = {
-  @val @scope(("window", "Dark", "fullstory")) external _setConsent: bool => unit = "setConsent"
+  // I assume this is some JS interop calling `window.Dark.fullstory`
+  // WHAT find the ReScript documentatino for this
+  @val
+  @scope(("window", "Dark", "fullstory"))
+  external _setConsent: bool => unit = "setConsent"
```

Comment:
There's an entire javascript interop section in the sidebar at https://rescript-lang.org/docs/manual/latest/external